### ISSUE TITLE
Use image descriptions set in Contentful as accessibility labels

### ISSUE THREAD

**Issue by brettdh** [goinverse-oss/app#196]
## Motivation
It's good practice to include an accessibility label for images that describes the content of the image. We should add these in Contentful and display them in the app.

To avoid duplicating the descriptions in multiple places that use the same image, we should use the "Description" field on an Asset in Contentful to store the accessibility label. This will require making a few changes:

- Import all images into Contentful and remove the external image URL fields
  - Though it's a bit of work, this will simplify the app and the CMS UI as well.
- Update the public podcast sync script to create assets as needed
  - This is a bit tricky, since items in Podbean feeds always contain an image
  - Should be able to compare the image URL in the item to the image URL of the feed, and only create assets for unique images

## Plan of action
- [ ] Ensure the description field is available via the API
- [ ] Import all external images to Contentful and remove external image fields
- [ ] Add descriptions to some images for testing
- [ ] Make the app add accessibility labels with those descriptions
- [ ] Add descriptions for the rest of the images
